server/src/main/java/io/spine/server/entity/storage/Column.java

```diff
@@ -108,8 +118,7 @@
  * <h2>Nullability</h2>
  *
  * <p>A Column may turn into {@code null} value if the getter which declares it is annotated as
- * {@link javax.annotation.Nullable javax.annotation.Nullable}. Otherwise, the Column is considered
- * non-null.
+ * {@link javax.annotation.Nullable}. Otherwise, the Column is considered non-null.
```

Comment:
Please revert this change. Spine Javadoc style checker would consider this as an error.

Reply:
Done.